tests/plugins/test_orchestrate_build.py

```diff
@@ -170,22 +171,43 @@ def make_worker_build_kwargs(**overrides):
     return kwargs
 
 
-def test_orchestrate_build(tmpdir):
+@pytest.mark.parametrize('worker_build_image', [
+    'fedora:latest',
+    None
+])
+def test_orchestrate_build(tmpdir, worker_build_image):
     workflow = mock_workflow(tmpdir)
     mock_osbs()
     mock_reactor_config(tmpdir)
+    plugin_args = {
+        'platforms': ['x86_64'],
+        'build_kwargs': make_worker_build_kwargs(),
+        'osbs_client_config': str(tmpdir),
+    }
+    if worker_build_image:
+        plugin_args['worker_build_image'] = worker_build_image
+
     runner = BuildStepPluginsRunner(
         workflow.builder.tasker,
         workflow,
         [{
             'name': OrchestrateBuildPlugin.key,
-            'args': {
-                'platforms': ['x86_64'],
-                'build_kwargs': make_worker_build_kwargs(),
-                'osbs_client_config': str(tmpdir),
-            }
+            'args': plugin_args
         }]
     )
+
+    plugin = runner.create_instance_from_plugin(OrchestrateBuildPlugin, plugin_args)
+    assert plugin.worker_build_image == worker_build_image
```

Comment:
This line is testing an aspect of the implementation, which we no longer need to do.